src/scenes/home/team/team.js

```diff
@@ -0,0 +1,95 @@
+import React, { Component } from 'react';
+import Section from 'shared/components/section/section';
+import TeamCard from 'shared/components/teamCard/teamCard';
+import StaffCard from 'shared/components/staffCard/staffCard';
+import armySeal from 'images/serviceSeals/Army.png';
+import navySeal from 'images/serviceSeals/Navy.png';
+import marineSeal from 'images/serviceSeals/USMC.png';
+import marineReserveSeal from 'images/serviceSeals/USMCReserve.png';
+import coastGuardSeal from 'images/serviceSeals/USCG.png';
+import airForceSeal from 'images/serviceSeals/AirForce.png';
+import airForceReserveSeal from 'images/serviceSeals/AirForceReserve.png';
+import armyReserveSeal from 'images/serviceSeals/ArmyReserve.png';
+import navyReserveSeal from 'images/serviceSeals/NavyReserve.png';
+import styles from './team.css';
+import Data from './team.json';
+
+class Team extends Component {
+  render() {
+    const founder = {
```

Comment:
This is fine for now, but lets create a ticket to move the data to a backend API.